workflow/rules/merge_fastq.smk

```diff
@@ -16,24 +16,24 @@ else:
     input = lambda wildcards: get_fastq_files(units, wildcards)
 
 
-rule zcat_fastq_files:
+rule decompress_and_merge_fastq_files:
```

Comment:
Why decompress and merge and not simply `cat` in a compressed state?

Reply:
Didn't know that you could do that! I will update it to not decompress

Reply:
done